portfolio_project7.html

```diff
@@ -0,0 +1,407 @@
+      <br>
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL9zygzz0hZalVlwxz9AsujtsrcMJpAP6S" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+	</div>
+    
+    <!-- Hide this text on small devices -->
+    <div class="w3-col m6 w3-hide-small w3-padding-large">
+      <h4 class=" w3-myfont">【向量圖繪製 及 加上文字及圖層 使用】</h4>
+        <h5 class=" w3-myfont">  ● </h5>
+		<h5 class="  w3-myfont"> ● </h5>
+		<h5 class="  w3-myfont"> ● </h5>	
+		<h5 class="  w3-myfont"> ●</h5>
+	<br>
+	</div>
+	
+	<div class="w3-col m6 w3-hide-small w3-padding-large">
+      <h4 class=" w3-myfont">【反覆播放的「動畫 」】</h4>
+        <h5 class=" w3-myfont">  ● </h5>
+		<h5 class="  w3-myfont"> ● </h5>
+		<h5 class="  w3-myfont"> ● </h5>	
+		<h5 class="  w3-myfont"> ●</h5>
+	<br>
+	</div>
+
+  </div>
+	
+	
+	<!--
+    <div class="w3-row w3-myfont"  id="proj4_photo">
+        <div class="w3-col m6 w3-center w3-padding-large">
+			<p><b><i class="fa fa-user w3-margin-right"></i>全線地圖</b></p>
+			<img src="pictures\media_archive_page\project_4_6_screenshot\mchouse1.jpg" onclick="onClick(this)" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Testing1" width="600" height="500">
+        </div>
+        <div class="w3-col m6 w3-center w3-padding-large">
+			<p><b><i class="fa fa-user w3-margin-right"></i>略版地圖</b></p>
+			<img src="pictures\media_archive_page\project_4_6_screenshot\mchouse3.jpg" onclick="onClick(this)" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Testing1" width="600" height="500">
+        </div>
+    </div>
+	<div class="w3-row w3-myfont"  id="proj4_photo">
+        <div class="w3-col m6 w3-center w3-padding-large">
+			<p><b><i class="fa fa-user w3-margin-right"></i>到達站台圖片</b></p>
+			<img src="pictures\media_archive_page\project_4_6_screenshot\mchouse2.png" onclick="onClick(this)" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Testing1" width="600" height="500">
+        </div>
+    <div class="w3-col m6 w3-center w3-padding-large">
+			<p><b><i class="fa fa-user w3-margin-right"></i>遊戲內 數值/範圍預覽</b></p>
+			<img src="pictures\media_archive_page\project_4_6_screenshot\mchouse4.png" onclick="onClick(this)" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Testing1" width="600" height="500">
+        </div>
+    </div>
+	
+-->
+
+
+  
+</div>
+
+  <div class="w3-row w3-center w3-dark-grey w3-padding-16">
+	<div class="w3-third w3-section w3-myfont"> <span class="w3-large">
+	    <br>
+		前往 圖片 (靜態) 作品示例
+		<br>
+		Picture archive
+		<br><br>
+		<a href="media_archive.html#galleries" class="buttonstyle_two"><i class="fa fa-arrow-right w3-margin-right"></i>Go 前往</a>
+			    <br>		<br>
+		前往 影片 (動態) 作品示例
+		<br>
+		Video archive<br><br>
+		<a href="media_archive.html#project_4_6" class="buttonstyle_two"><i class="fa fa-arrow-right w3-margin-right"></i>Go 前往</a>

```

Comment:
### Add the following external support on `media` page

- [ ] https://youtu.be/KWNnBKXlFaQ